medic/medic-appium.js

```diff
@@ -355,9 +358,8 @@ function isFailFastError(error) {
 function killProcess(procObj, killSignal, callback) {
     if (procObj.alive) {
         procObj.alive = false;
-        setTimeout(function () {
-            kill(procObj.process.pid, killSignal, callback);
-        }, 1000);
```

Comment:
As far as I remember, these process-killing timeouts were needed to give the treekill module some time to actually kill the process. On mac it may be fine, but on Windows the Appium process is left hanging in the memory and occupying the port.